Add replacement "\sinh(x)" -> "\sinh@{x}"

poortho opened the pull request:
Adds code to change \sinh(x) to \sinh@{x}.
Also added test cases for it and made it so that the output when running bin/texvcjs ends with a newline

poortho committed:
Add replacement for "\sinh(x)" -> "\sinh@{x}"

Close #4

poortho committed:
Merge branch 'master' of github.com:ClaudeZou/texvcjs

Conflicts:
	test/all.js

physikerwelt commented on bin/texvcjs:
```diff
@@ -45,10 +45,10 @@ PACKAGES.forEach(function(pkg) {
 // output result
 if (result.status === '+') {
-    util.print(result.status + (result.output || ''));
+    console.log(result.status + (result.output || ''));
```
why did you change that?

poortho replied:
This makes it so that there is a new line after the output. 
Previously, ./texvcjs "\sin" would return

```
+\sin
```

but now it returns

```
+\sin

```

This makes it so that in terminal the output and the prompt for the next command line input are on different lines.

physikerwelt replied:
ok

physikerwelt replied:
Can you undo this change. I think we should remove the emptyline or at least make this a configurable option. Especially when the output is consumed by other programs this might be a problem.

physikerwelt commented on lib/astutil.js:
```diff
@@ -103,6 +103,10 @@ ast.Tex.defineVisitor("contains_func", {
         return r.tex_contains_func(target);
     },
+    PAREN: function(target, f, a) {
+        // {\f@{a}}
```
make a comment similar to line 111. people usually don't have a clue what @ stands for

poortho replied:
Do you mean something like this?

```
// {\f@{a}} (function of one argument with an @ symbol to indicate how it should be rendered) 
```

physikerwelt replied:
Yes.

poortho committed:
Fix bugs for \sinh(x) -> \sinh@{x}

poortho committed:
Remove newline character from output

poortho committed:
Merge branch 'master' of github.com:DRMF/texvcjs

poortho commented:
I updated this pull request just now. It no longer has a new line at the end of the output (as requested by Moritz) and I also fixed to following bugs in the previous commit regarding the \sinh(x) -> \sinh@{x} replacement.
-not accepting multiple literals (for example, it would throw an error when the input was "\sinh(xyz)")
-not accepting itself as a valid argument
-not accepting empty arguments inside the parenthesis
-not accepted as an argument for other LaTeX commands

physikerwelt commented:
@ClaudeZou I would not call it bugs in texvcjs ... those "bugs" only occour since we changed the functionality and the purpose of texvcjs

poortho commented:
@physikerwelt Ok, I changed it to bugs in the previous commit, is that acceptable?

physikerwelt commented:
@ClaudeZou it's not that essential it's just a side remark
